kornia/geometry/conversions.py

```diff
@@ -1175,3 +1185,163 @@ def normalize_homography3d(
     # compute chain transformations
     dst_norm_trans_src_norm: torch.Tensor = dst_norm_trans_dst_pix @ (dst_pix_trans_src_pix @ src_pix_trans_src_norm)
     return dst_norm_trans_src_norm
+
+
+def extrinsics_from_Rt(R: Tensor, t: Tensor) -> Tensor:
+    r"""Combines 3x3 R and 1x3 t into 4x4 extrinsics.
+    Extrinsics matrix is usually named `K`, but we avoid this,
```

Comment:
I do not understand this comment here, is it relevant for the user? moreover, as far as I know are the intrinsic teh ones that tipically are called K, right?

For me this method is not computing the extrinsics, you already have them (R and t), what you are doing is putting them together into a 4x4 matrix, right?

Reply:
Yes, it got mixed in my head, you are right - K is calibration.

Reply:
@lferraz @edgarriba how about `combine_Rt` and `split_Rt` instead?

Reply:
`matrix_from_rt` // `rt_from_matrix` ?

Reply:
`matrix4d_from_rt`? Xd

Reply:
`matrix4x4_from_rt`?